### PR TITLE
Feat/dev 1637 fix bug in indexd utils

### PR DESCRIPTION
**Pull request by qiaouchicago**
fix indexd utils

have to pass params from setup_database to try_drop_indexd_database so we can pass host for postgres db on different host.

### CODE REVIEW

File: indexd/utils.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def setup_database(
     """Setup the user and database"""
 
     if not no_drop:
-        try_drop_test_data(database)
+        try_drop_test_data(database=database, root_user=root_user, host=host)
```
Comment (XukunXiangUChicago) [inline indexd/utils.py:50]: I think I'm missing something: does this change anything? The `setup_database` gets called [here|https://github.dev/NCI-GDC/indexd/blob/5932bddbc27048855e55735e1522f08cadd50222/tests/conftest.py#L39] without any parameters, are we planning to provide `root_user` and `host` in the future?

Comment (qiaouchicago) [reply indexd/utils.py:50]: The utils of indexd is not only used by tests of indexd, it is currently also used by most repos that needs indexd server for tests.

File: indexd/default_settings.py [hunk 2]
```diff
@@ -11,7 +11,8 @@
 SQLALCHEMY_VERBOSE = (
     os.getenv('INDEXD_VERBOSE', '').lower() in ['1', 'yes', 'true']
 )
-PG_URL = 'postgresql://test:test@localhost/indexd_test'
+PG_HOST = os.getenv("PG_INDEXD_HOST", "localhost")
```
Comment (stilesj-uchicago) [inline indexd/default_settings.py:14]: All the way in here and didn't want to extract the user (test) and password (test) to environment variables?

**Review by stilesj-uchicago — APPROVED** [review]

File: indexd/default_settings.py (hunk 2 above)
Comment (qiaouchicago) [reply indexd/default_settings.py:14]: LOL, sorry  I was in a hurry to solve my problem and did not notice that.

**Pull request merged**